app.js

```diff
@@ -124,6 +130,13 @@ const authorize = (proof, stats) => {
   return isAuthorized
 }
 
+api.on('error', function error(err) {
+  console.error('Something horrible has happened', err.stack);
+});
+
+api.on('reconnect', function (opts) {
+  console.error('Reconnection attempt started', opts);
```

Comment:
Is this an error? Also not sure how logging with this guy works but if we really care winston or something might be nice

Reply:
logging works by passing the `VERBOSITY` env var
0 is none, 1 is error level, 2 is more